addons/block_code/types/types.gd

```diff
@@ -12,14 +12,16 @@ enum BlockType {
 	VECTOR2,
 	BOOL,
 	COLOR,
-	NODE
+	NODE,
+	OPTION
```

Comment:
Sure, I was just reusing the parameter input generation we already had, because it ties into the code generation, and it basically is just a parameter input with some default text values. I suppose I can implement another component for this and add different syntax to create them in the `StatementBlock` formatting code if you think that's better.